src/scripts/genEtwProvider.py

```diff
@@ -22,7 +22,8 @@
 etw_dirname = "etw"
 replacements = [
     (r"EventEnabled", "EventXplatEnabled"),
-    (r"\bPVOID\b", "void*")
+    (r"\bPVOID\b", "void*"),
+    (r"_Arg0 \* _Arg2_Len_", "_Arg2_Len_")
```

Comment:
Since we are relying on the non-guaranteed invariant that this line occurs only once in the generated code (by mc.exe) we should add something in the script below that asserts / fails when there are two or more of such line so that we can at least try to find another "surgical" fix to this issue.